presto-docs/src/main/sphinx/release/release-340.md

```diff
@@ -0,0 +1,75 @@
+# Release 340 (XX Aug 2020)
+
+## General Changes
+
+* Add support for query parameters in `LIMIT`, `OFFSET` and `FETCH FIRST` clauses. ({issue}`4529`, {issue}`4601`)
+* Add experimental support for recursive queries. ({issue}`4250`)
+* Add {func}`bitwise_left_shift`, {func}`bitwise_right_shift` and {func}`bitwise_right_shift_arithmetic`. ({issue}`772`)
+* Add {func}`luhn_check`. ({issue}`4011`)
+* Add `IF EXISTS `and `IF NOT EXISTS` syntax to `ALTER TABLE`. ({issue}`4651`)
+* Include remote host in error info for page transport errors. ({issue}`4511`)
+* Enable HTTP/2 by default for internal communication. This can be disabled using the
+  `internal-communication.http2.enabled` configuration property. ({issue}`4696`)
+* Improve minimum latency for dynamic partition pruning. ({issue}`4388`}
+* Reduce cluster load by cancelling query stages from which data is no longer required. ({issue}`4290`)
+* Reduce query memory usage by improving retained size estimation for `VARCHAR` and `CHAR` types. ({issue}`4123`)
+* Improve query performance for queries containing {func}`starts_with`. ({issue}`4669`)
+* Improve performance of queries that use `DECIMAL` data type. ({issue}`4730`)
+* Fix failure when `GROUP BY` clause contains duplicate expressions. ({issue}`4609`)
+* Fix potential hang during query planning ({issue}`4635`).
+
+## Security Changes
+
+* Fix unprivileged access to table's schema via `CREATE TABLE LIKE`. ({issue}`4472`)
+
+## JDBC Driver Changes
+
+* Fix handling of dates and timestamps before the 20th century. ({issue}`4563`)
```

Comment:
Is this redundant with the two items below?

Reply:
Yes, forgot to remove the old one.